plugins/techdocs-module-addons-contrib/src/ReportIssue/ReportIssue.tsx

```diff
@@ -118,7 +118,12 @@ export const ReportIssueAddon = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [selection, mainContent, feedbackContainer]);
 
-  if (!selection || !repository) return null;
+  if (
+    !selection ||
+    !repository ||
+    !['github', 'gitlab'].includes(repository.type)
```

Comment:
Instead of explicitly including all the others (allow list), maybe it better to invert this (block list) so it only needs to be adjusted if some integration does not support it?

And what is the reason for not supporting the ReportIssueAddon?

Reply:
> And what is the reason for not supporting the ReportIssueAddon?
If I'm not wrong, the `IssueLink` component doesn't know how to create the URL for bitbuckt here:
https://github.com/backstage/backstage/blob/ad35364e971bf21d26e7b8ec3d74a00323996268/plugins/techdocs-module-addons-contrib/src/ReportIssue/IssueLink.tsx#L66-L69

Reply:
Bitbucket Server doesn't have an integrated Issue Tracker system. Therefore `ReportIssueAddon` is not applicable here.

Reading the method below, I assumed that `ReportIssueAddon` was developed to support only Github and Gitlab. That's why I added the condition that way.

https://github.com/backstage/backstage/blob/ad35364e971bf21d26e7b8ec3d74a00323996268/plugins/techdocs-module-addons-contrib/src/ReportIssue/IssueLink.tsx#L50-L55

Reply:
I see. Thanks for the clarification. Bitbucket Cloud would have an integrated issue tracker, but I guess many use Jira instead though which got more integrated there as well some time ago.

The allow list makes perfect sense then!